languages/java/exercises/concept/constructors/.docs/introduction.md

```diff
@@ -0,0 +1,36 @@
+Creating an instance of a _class_ is done by calling its _constructor_ through the `new` operator. A constructor is a special type of method whose goal is to initialize a newly created instance. Constructors look like regular methods, but without a return type and with a name that matches the classes' name.
+
+```java
+class Library
+{
+    private final int books;
```

Comment:
Are students familiar with the `final` modifier? Only language features introduced by the exercise or its prerequisites (and their prerequisites, and so on) should be used. This could mean that in some cases the code might not be completely idiomatic.

Reply:
Removed final in the intro as well.